src/perfglue/heap_profiler.cc

```diff
@@ -153,6 +163,19 @@ void ceph_heap_profiler_handle_command(const std::vector<std::string>& cmd,
   } else if (cmd.size() == 1 && cmd[0] == "release") {
     ceph_heap_release_free_memory();
     out << g_conf()->name << " releasing free RAM back to system.";
+  } else if (cmd.size() == 1 && cmd[0] == "get_release_rate") {
+    out << g_conf()->name << " release rate: " 
+	<< std::setprecision(4) << ceph_heap_get_release_rate() << "\n";
+  } else if (cmd.size() == 2 && cmd[0] == "set_release_rate") {
+    std::string err;
+    double val = strict_strtod(cmd[1].c_str(), &err);
+    if (err.empty()) {
+      ceph_heap_set_release_rate(val);
+      out << g_conf()->name <<  " release rate changed to: " 
+          << std::setprecision(4) << ceph_heap_get_release_rate() << "\n";
+    } else {
+      out << g_conf()->name <<  " *** need an numerical value.";
```

Comment:
nit, if you don't care about the **content** of `err`, why bothering using `strict_strtod()`? i mean, `std:: stod()` would suffice here.